### DISCUSSION TITLE
我老婆好康吧

### DISCUSSION THREAD

**Discussion by Futao-Augenstern** [share121/inter-knot#2177] [其他]
![1000154078](https://github.com/user-attachments/assets/2ca8442c-d5d7-4fb9-9235-33e51e00baec)

**Comment by MissHedwig** [comment]
这是我老婆

**Reply by Futao-Augenstern** [reply]
俺的

**Comment by Hu-tang-zhu** [comment]
<img src="https://bbs-static.miyoushe.com/static/2023/10/18/ce972f0fa05e054e92ea76bc8be8a630_1414225707560137048.png" alt="胡桃-咘咘！不对！" width="75" ><img src="https://img-static.mihoyo.com/communityweb/upload/fe198bb1523528c2b852d28626379a2a.png" alt="心情简单" width="75" >这是我家的！<img src="https://img-static.mihoyo.com/communityweb/upload/ee7bb800b0c90f66fa9e321cb6d8ee17.png" alt="抱大腿" width="75" >

**Reply by Futao-Augenstern** [reply]
俺家的

**Comment by NOVAQUQ** [comment]
![1701142776402386f93da8a2438f8527ccfe1c90c4c0f1e54820ff0610e0b67187f778f7bfa7b 0](https://github.com/user-attachments/assets/65242243-fbf6-4537-b04d-e633f7c0c05a)
∠( ᐛ 」∠)_

**Reply by JobNumber-9527** [reply]
无鸡之谈

**Reply by shiliuyexiafeng** [reply]
> 无鸡之谈

磨豆腐不行吗🌝

**Comment by satokuoye** [comment]
好好好
![GUC93Z9acAA175p](https://github.com/user-attachments/assets/e5b4952a-2673-488e-a2b0-e1761bf9c048)

**Comment by 1576655988** [comment]
![img_-1324161296_1704017718875](https://github.com/user-attachments/assets/340dda3e-7820-456e-92c2-dc5210b54b15)
<img src="https://img-static.mihoyo.com/communityweb/upload/6744fdc365a234f152bc3fcc6443c7a9.png" alt="米游姬糖葫芦" width="75" >

**Reply by Yoimiya-M** [reply]
什么远古老图

**Comment by yinghe7banjiangyang** [comment]
我的{超大声}
<img src="https://img-static.mihoyo.com/communityweb/upload/b359242ee7d6676fc02afc68552eb311.png" alt="胡桃-吃惊" width="75" >

**Comment by elysiam2** [comment]
什么你的，这是我的

**Comment by YuTinMIn** [comment]
真豪康（赞赏）

**Comment by JobNumber-9527** [comment]
什么泥的，这明明是窝的

**Comment by qiuyezhi** [comment]
你们都不要抢了，这是我老婆

**Comment by SCJWY** [comment]
我老婆！！

**Comment by zzz-Rope** [comment]
我的！<img src="https://bbs-static.miyoushe.com/static/2023/07/17/ddde76ee17369663665f077dd135bd69_9187505232048012157.png" alt="珊瑚宫心海-思考" width="75" >

**Comment by famgming** [comment]
好看

**Comment by YSYYXS** [comment]
他真好看

**Comment by adsadk12** [comment]
好看

**Comment by Amateur-112** [comment]
她真好看

**Comment by fyhwsy** [comment]
可以的
